autoload/lsp/ui/vim/output.vim

```diff
@@ -2,6 +2,11 @@ let s:use_vim_popup = has('patch-8.1.1517') && g:lsp_preview_float && !has('nvim
 let s:use_nvim_float = exists('*nvim_open_win') && g:lsp_preview_float && has('nvim')
 let s:use_preview = !s:use_vim_popup && !s:use_nvim_float
 
+let s:Markdown = vital#lsp#import('VS.Vim.Syntax.Markdown')
+let s:MarkupContent = vital#lsp#import('VS.LSP.MarkupContent')
+let s:Window = vital#lsp#import('VS.Vim.Window')
+let s:Text = vital#lsp#import('VS.LSP.Text')
+
```

Comment:
I'm worrying if it hit the performance or not.

I will investigate it.

Reply:
do u mean performance of import? If that is the case we could check if `s:Markdown` exists and then only load it else don't load it at all.

or do u mean perf or `Markdown.apply`? or `MarkupContent.normalize`?

Reply:
Yes. I mean the performance of import.

The Markdown.{apply,normalize} is not lightweight but we can't avoid it, I think.

Reply:
sounds good. lazy loaded markdown in the meantime.